eng/pipelines/weekly_automation.yml

```
@@ -26,7 +26,7 @@ jobs:
       BaseBranchName: $(DefaultBranch)
       PRBranchName: automated-rush-update-full
       CommitMsg: "Automatic rush update --full"
-      PRTitle: "Automatic rush update --full"
+      PRTitle: "[EngSys] automatic rush update --full"
```

```suggestion
      PRTitle: "[Automation] rush update --full"
```
I think I would prefer a tag like [Automation] over [EngSys] as EngSys leads me to think the EngSys team needs to take action on this PR.

---

We have been using `EngSys` label for JS engsys work items and `Central EngSys` for EngSys team work items. I prefer EngSys because it reflects category of changes that are not directly related to services SDK product code.

---

OK as long as the implications aren't that the EngSys team needs to take action on all these I'm fine with that.